src/python/pants/option/options.py

```diff
@@ -317,14 +317,25 @@ def for_scope(self, scope, inherit_from_enclosing_scope=True):
 
     return values
 
-  def get_fingerprintable_for_scope(self, scope):
+  def get_fingerprintable_for_scope(self, scope, include_passthru=False):
```

Comment:
Where is there behavior change? (beyond the change you're deliberately introducing)